js_4/hm6Task2.js

```diff
@@ -7,24 +7,20 @@
 // const competitorsPizzas = ['Peperoni', 'Caprichosa', 'Diablo', '4 cheeses', 'hawai']
 
 const competitorsPizzas = ['Peperoni', 'Caprichosa', 'Diablo', '4 cheeses', 'hawai']
-const myPizza = ['peperoni', 'caprichosa', '4 cheeses']
-// const myPizza = ['peperoni', 'caprichosa', 'diablo', '4 cheeses', 'hawai']
-checkPizzasWithCompetitors(myPizza)
+
+// const myPizzas = ['peperoni', 'caprichosa', '4 cheeses']
+const myPizzas = ['peperoni', 'caprichosa', 'diablo', '4 cheeses', 'hawai']
+
+checkPizzasWithCompetitors(myPizzas)
 
 function checkPizzasWithCompetitors(myPizzas){
```

Comment:
- Названия параметров функции не делаем, как названия данньіх с которьіми будем работать, дабьі избежать возможньіх колизий
```suggestion
function checkPizzasWithCompetitors(arr) {
```
- Так же на вход функции можем подавать `два` массива и сравнивать их

Reply:
исправила